src/Raven.Server/Documents/Indexes/Persistence/Corax/CoraxDocumentConverter.cs

```diff
@@ -0,0 +1,255 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using Corax;
+using Raven.Client;
+using Raven.Server.Json;
+using Raven.Server.Utils;
+using Sparrow;
+using Sparrow.Extensions;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+using Sparrow.Server;
+using Sparrow.Threading;
+using Voron;
+
+namespace Raven.Server.Documents.Indexes.Persistence.Corax
+{
+    public sealed class CoraxDocumentConverter : ConverterBase, IDisposable
+    {
+        private readonly BlittableJsonTraverser _blittableTraverser;
+        private readonly Dictionary<string, IndexField> _fields;
+        private readonly Index _index;
+        private readonly ByteStringContext _allocator;
+        private readonly Dictionary<Slice, int> _knownFields;
+        private static readonly string _trueLiteral = "true";
+        private static readonly string _falseLiteral = "false";
```

Comment:
Changed into:
`private static readonly byte[] _trueLiteral = Encoding.UTF8.GetBytes("true"); 
`
It would be faster than changing it every time from a string into bytes. I can make it const but this required writing literals already converted to byte arrays.

Reply:
No need, this will be turned into a real const by the tiered JIT

Reply:
Not that, for that to happen you need to use:
`private static readonly ReadOnlySpan<byte> _trueLiteral = new ReadOnlySpan<byte>(Encoding.UTF8.GetBytes("true")`

Reply:
This is a class and I got:
`Field cannot be of byref-like type 'System.ReadOnlySpan<byte>' unless it is an instance member of a 'ref' struct`